application-analytics-ui/src/main/resources/Analytics/Code/Macros/RowEvolution.xml

```diff
@@ -146,15 +146,30 @@ require(['jquery', 'chart'], ($) =&gt; {
   let currentModal;
   let currentOptions;
   let currentLoadingAnimation;
-
+  /**
+  * Processes the data parameter and returns an array of numbers.
+  *
+  * @param data json with the dataset.
+  * @param field the field that will pe processed.
+  */
+  function processChartValues(data, field) {
+      let result = $.map(data, function(currentItem) {
```

Comment:
This isn't supposed to be indented. The next lines are because they are inside of the callback function